### ISSUE TITLE
Add upvote and downvote

### ISSUE THREAD

**Issue by Mrancois** [wktang/nodebb-plugin-category-sort-by-votes#3]
Is it possible to add upvote and downvote on main post beside votes ?

**Comment by xidui** [comment]
You mean add "sort by upvote" and "sort by downvote"? 
Yeah, technically feasible. Just making another two index will be fine.
@wktang what do you think about this feature?

**Comment by Mrancois** [comment]
No, I mean add the **button** upvote and downvote

**Comment by xidui** [comment]
I think [this](https://discuss.leetcode.com/topic/207/water-and-jug-problem) may be what you want. :)

**Comment by wktang** [comment]
@Mrancois Correct me if I'm wrong, I think @Mrancois means adding the button for each topic in the question list:

Something like this (You can upvote/downvote from the question list page):
![image](https://cloud.githubusercontent.com/assets/2770875/16252853/78f21092-37e6-11e6-9631-edcaecce2b43.png)

**Comment by xidui** [comment]
@wktang 
If user only see the title, how should he vote without any knowledge of the topic?

**Comment by xidui** [comment]
@Mrancois 
If you think this feature is really useful, feel free to send a PR to us.
It is better if you add a switch in the setting page, thus site admin can disable it or enable it according to their own tastes.

**Comment by wktang** [comment]
@xidui Yes, even StackOverflow does not have the vote button in question list view, although some users may need it for their own purpose.

@Mrancois If you want you can implement this feature and send a PR to us, as we currently do not have the need to implement this.
